src/Collapse.js

```diff
@@ -117,10 +120,11 @@ class Collapse extends Component {
           let collapseClass = getTransitionClass(status);
           const classes = mapToCssModules(classNames(
             className,
+            horizontal && 'collapse-horizontal',
             collapseClass,
             navbar && 'navbar-collapse'
           ), cssModule);
-          const style = height === null ? null : { height };
+          const style = dimension === null ? null : { [horizontal ? 'width' : 'height']: dimension };
```

Comment:
Can we add a test that the actual style is set correctly depending on whether the `horizontal` prop is true or false?

Reply:
Good idea, however this is not easy to test as the style attribute is removed after transition is over - the height and width are only present during the transition.

Reply:
Gotcha - that's fine then!